web/tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "jsx": "preserve",
     "types": ["vite/client"]
   },
-  "include": ["./src"]
+  "include": ["./src", "."]
```

Comment:
You shouldn't need this as the original setting just forces our prod code to be checked rather than the other code.
Was this to solve some issue locally for you?

Reply:
Yeah without that I was getting import errors in playwright config